worlds/raft/__init__.py

```diff
@@ -129,15 +146,15 @@ def collect_item(self, state, item, remove=False):
         return super(RaftWorld, self).collect_item(state, item, remove)
 
     def pre_fill(self):
-        if self.multiworld.island_frequency_locations[self.player] == 0:
+        if self.multiworld.island_frequency_locations[self.player] == 0: # Vanilla
```

Comment:
```suggestion
        if self.multiworld.island_frequency_locations[self.player] == "vanilla":
```
for future reference, you can just do that.